### DISCUSSION TITLE
Custom Firmware 2022.01.24?

### DISCUSSION THREAD

**Discussion by dmavromatis** [ludwig-v/wireless-carplay-dongle-reverse-engineering#132] [General]
@ludwig-v let me know when you have the custom firmware of 2022.01.24 ready for us.  Thanks!

**Comment by dmavromatis** [comment]
@ludwig-v is it possible to get some latest version of firmware released as custom?  Or if there is a documented process on how we can do that ourselves?

**Comment by ghost** [comment]
Also waiting for this…. Ludwig can you assist?
